numpy/random/mtrand/mtrand.pyx

```diff
@@ -2865,16 +2865,16 @@ cdef class RandomState:
 
         fa = PyFloat_AsDouble(a)
         if not PyErr_Occurred():
-            if fa <= 0:
-                raise ValueError("a <= 0")
+            if fa < 0:
+                raise ValueError("a < 0")
             return cont1_array_sc(self.internal_state, rk_power, size, fa,
                                   self.lock)
 
         PyErr_Clear()
 
         oa = <ndarray>PyArray_FROM_OTF(a, NPY_DOUBLE, NPY_ARRAY_ALIGNED)
-        if np.any(np.less_equal(oa, 0.0)):
-            raise ValueError("a <= 0")
+        if np.any(np.less(oa, 0.0)):
+            raise ValueError("a < 0")
         return cont1_array(self.internal_state, rk_power, size, oa, self.lock)
```

Comment:
[And another](https://github.com/numpy/numpy/blob/master/numpy/random/mtrand/distributions.c#L629).